components/sysutils/sudo/Makefile

```diff
@@ -30,15 +30,15 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		sudo
 SRC_VERSION=		1.9.7
-SRC_PATCH_VERSION=	0
+SRC_PATCH_VERSION=	2
 SRC_DOT_PATCH_VERSION=	$(shell [ $(SRC_PATCH_VERSION) -gt 0 ] && echo .$(SRC_PATCH_VERSION))
```

Comment:
Additionally you can also supply HUMAN_VERSION here and in the manifest.

Reply:
Did it, thanks.